pkg/kubectl/cmd/patch.go

```diff
@@ -36,13 +37,16 @@ Please refer to the models in https://htmlpreview.github.io/?https://github.com/
 // Partially update a node using strategic merge patch
 kubectl patch node k8s-node-1 -p '{"spec":{"unschedulable":true}}'
 
+// Partially update a node from node.json using strategic merge patch
```

Comment:
I am in favor of this but currently all of the example comments use `//` so lets stay consistent.